Slim/Menu/BrowseLibrary/Releases.pm

```diff
@@ -102,6 +124,10 @@ sub _releases {
 			# don't list as trackartist, if the artist is albumartist, too
 			next if $role eq 'TRACKARTIST' && $isPrimaryArtist{$_->{id}};
 
+			if ( $role eq 'COMPOSER' && ( $genreMatch || $allComposers ) ) {
```

Comment:
Very nice cleanup you've done here with the helper variables! Makes things so much more readable.